io.github.electronstudio.WeylusCommunityEdition.json

```diff
@@ -0,0 +1,84 @@
+{
+  "id": "io.github.electronstudio.WeylusCommunityEdition",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.rust-stable",
+    "org.freedesktop.Sdk.Extension.node18"
+  ],
+  "command": "weylus",
+  "finish-args": [
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--share=ipc",
+    "--device=all",
+    "--share=network",
+    "--filesystem=xdg-run/pipewire-0"
+  ],
+  "modules": [
+    {
+      "name": "x264",
+      "buildsystem": "autotools",
+      "config-opts": [
+        "--prefix=/app","--exec-prefix=/app","--disable-opencl","--disable-cli","--enable-shared"
+      ],
```

Comment:
```suggestion
      "config-opts": [
        "--exec-prefix=/app",
        "--disable-opencl",
        "--disable-cli",
        "--enable-shared"
      ],
```

Reply:
Long lines will get overwritten if you use x-checker also prefix is automatically set.

Reply:
I'm pretty sure `--exec-prefix` is also not necessary.